pkg/api/api.go

```diff
@@ -350,36 +350,58 @@ func failureGroupList(report util.TestReport) []FailureGroup {
 	return failureGroups
 }
 
-// PrintJSONReport prints the json format of the report
-// follows conventions from jsonapi.org
-func PrintJSONReport(w http.ResponseWriter, req *http.Request, report, prevReport util.TestReport, endDay, jobTestCount int) {
-
+func formatJSONReport(report, prevReport util.TestReport, endDay, jobTestCount int) map[string]interface{} {
 	data := html.TestReports{
 		Current:      report,
 		Prev:         prevReport,
 		EndDay:       endDay,
 		JobTestCount: jobTestCount,
 		Release:      report.Release}
+	// NOTE: should I use report.Release or pass in release from main.go
```

Comment:
since you have to pass the report object in anyway, this seems reasonable rather than adding a redundant arg